problemsets-mat223/linearalbegra.tex

```diff
@@ -2353,56 +2353,70 @@ \section*{Subspaces and Bases}
-					comes out.
+				\item Students will fail to distinguish $\Span\{\vec u,\vec v\}$ 
+					from $\{\vec u,\vec v\}$. Make sure this distinction comes out.
 			\end{itemize}
 		\end{notes}
 	\end{annotation}
-	Let $\vec u=\mat{1\\0\\0}$, $\vec v=\mat{0\\1\\0}$, $\vec w=\mat{1\\1\\0}$, and $V=\Span\{\vec u,\vec v,\vec w\}$.
+	Let $\vec u=\mat{1\\0\\0}$, $\vec v=\mat{0\\1\\0}$, $\vec w=\mat{1\\1\\0}$, 
+	and $V=\Span\Set{\vec u,\vec v,\vec w}$.
 	\begin{parts}
 		\item Describe $V$.
-			\begin{solution}
+			\begin{solution}[inline]
 				$V$ is the $xy$-plane in $\R^3$.
 			\end{solution}
-		\item Is $\{\vec u,\vec v,\vec w\}$ a basis for $V$?  Why or why not?
+		\item Is $\Set{\vec u,\vec v,\vec w}$ a basis for $V$?  Why or why not?
 			\begin{solution}
-				No. The set $\{\vec u,\vec v,\vec w\}$ is linearly dependent since $\vec w=\vec u+\vec v$.
+				No. The set $\Set{\vec u,\vec v,\vec w}$ is linearly dependent since $\vec w=\vec u+\vec v$.
 			\end{solution}
 		\item Give a basis for $V$.
-			\begin{solution}
-				$\{\vec u,\vec v\}$
+			\begin{solution}[inline]
+				$\Set{\vec u,\vec v}$.
 			\end{solution}
 		\item Give another basis for $V$.
-			\begin{solution}
-				$\{\vec u,\vec w\}$ or $\{\vec v,\vec w\}$.
+			\begin{solution}[inline]
+				$\Set{\vec u,\vec w}$ or $\Set{\vec v,\vec w}$.
 			\end{solution}
-		\item Is $\Span\{\vec u,\vec v\}$ a basis for $V$?  Why or why not?
+		\item Is $\Span\Set{\vec u,\vec v}$ a basis for $V$?  Why or why not?
 			\begin{solution}
-				No. $\Span\{\vec u,\vec v\}$ is an infinite set of vectors
+				No. $\Span\Set{\vec u,\vec v}$ is an infinite set of vectors
 				which includes $\vec 0$, so it cannot be linearly independent and
 				therefore isn't a basis.
 			\end{solution}
 		\item What is the dimension of $V$?
 			\begin{solution}
-				A basis for $V$ has two vectors so it is two dimensional. We also
-				know this because $V$ is the $xy$-plane in $\R^3$ and all planes are two
-				dimensional.
+				A basis for $V$ has two vectors so it is two-dimensional. We also
+				know this because $V$ is the $xy$-plane in $\R^3$ and all planes 
+				are two-dimensional.
 			\end{solution}
 	\end{parts}
 
 
 	\question
 	Let $\vec a=\mat{1\\2\\3}$, $\vec b=\mat{4\\5\\6}$, $\vec c=\mat{7\\8\\8}$ (notice these vectors
 	are linearly independent) and 
-	let $P=\Span\{\vec a,\vec b\}$ and $Q=\Span\{\vec b,\vec c\}$.
+	let $P=\Span\Set{\vec a,\vec b}$ and $Q=\Span\Set{\vec b,\vec c}$.
 	\begin{parts}
 		\item Give a basis for and the dimension of $P$.
+			\begin{solution}
+				$\Set{\vec a,\vec b}$ is a basis for $P$, and so its dimension is 2. 
+			\end{solution}
 		\item Give a basis for and the dimension of $Q$.
+			\begin{solution}
+				$\Set{\vec b,\vec c}$ is a basis for $Q$, and so its dimension is 2. 
+			\end{solution}
 		\item Is $P\cap Q$ a subspace? If so, give a basis for it and its dimension.
+			\begin{solution}
+				Yes. $\Set{\vec b}$ is a basis for $P\cap Q$, and so its dimension is 1. 
```

Comment:
Sure. I'm writing out an explanation that essentially proves that the intersection of any two subspaces is a subspace. Would you prefer I do something more specific to this case? Like I could just prove that P \cap Q = span{b} and conclude everything from that.

Reply:
I'm kind of doing both, as you'll see. I expect you'll want me to keep only one half of this. I'll wait for your word.

Reply:
You can just say two non-parallel planes intersect in a line. Since c and 0 are common to both planes, the line must contain c and 0 and is therefore span{c}

Reply:
That's much easier.